src/main/kotlin/study/racingcar/step3/ParticipationCar.kt

```diff
@@ -0,0 +1,10 @@
+package study.racingcar.step3
+
+class ParticipationCar(private val participationCount: Int) {
```

Comment:
ParticipationCar 클래스는 `cars: List<Car>`를 인스턴스 변수로 가지고 있습니다.

명확한 의미를 드러내기 위해 아래와 같이 클래스명을 변경하면 어떨까요?

```suggestion
class ParticipationCars(private val participationCount: Int) {
```